backend/src/main/java/edu/netcracker/backend/dao/impl/TicketDAOImpl.java

```diff
@@ -1,16 +1,17 @@
 package edu.netcracker.backend.dao.impl;
 
-import edu.netcracker.backend.dao.CrudDAO;
 import edu.netcracker.backend.dao.TicketDAO;
 import edu.netcracker.backend.model.Ticket;
+import lombok.extern.log4j.Log4j2;
 import org.springframework.dao.EmptyResultDataAccessException;
 import org.springframework.stereotype.Repository;
 
 import java.util.ArrayList;
 import java.util.List;
 
 @Repository
-public class TicketDAOImpl extends CrudDAO<Ticket> implements TicketDAO {
+@Log4j2
+public class TicketDAOImpl extends CrudDAOImpl<Ticket> implements TicketDAO {
 
     private final String findAllByClass = "SELECT * FROM ticket WHERE class_id = ?";
```

Comment:
1) list all columns which would you like to get even if you need to get all columns
2) relocate this query to enum or sql file